### DISCUSSION TITLE
Apply failure on Github Education

### DISCUSSION THREAD

**Discussion by sonnaWater** [community/community#53301] [GitHub Education] [answered]
### Select Topic Area

Question

### Body

hi，

I have upload my school certificate of enrollment to you for verification, but I don't know why I receive the following response,
"""
You are unlikely to be verified until you have completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with your full name as it appears in your academic affiliation document plus a short bio.

Please consider using your device camera to submit academic affiliation documents. Uploaded images are more easily manipulated and are therefore less trustworthy.
"""

thanks for your help.

![image](https://user-images.githubusercontent.com/120314510/233371862-3e9b1238-7534-42d2-b154-60cfb53efd5f.png)

**Comment by rmdhfz** [comment, marked as answer]
Hi @sonnaWater ,

It seems that the verification process requires you to complete your GitHub user profile with your full name as it appears in your academic affiliation document and a short bio. Please ensure that your profile is complete with the required information before resubmitting your academic affiliation document.

Additionally, the message suggests using your device camera to submit the document instead of uploading an image file. This is because uploaded images can be easily manipulated, and using a device camera can provide a more trustworthy verification process.
